ResearchKit/Common/ORKDefines_Private.h

```diff
@@ -37,8 +37,9 @@ ORK_EXTERN NSBundle *ORKBundle() ORK_AVAILABLE_DECL;
 #define ORKLocalizedString(key, comment) \
 [ORKBundle() localizedStringForKey:(key) value:@"" table:nil]
 
-#define ORKDynamicCast(x, c) ((c *) ([x isKindOfClass:[c class]] ? x : nil))
+ORK_EXTERN id _ORKDynamicCast(id x, Class objClass) ORK_AVAILABLE_DECL;
```

Comment:
Let's rename to put the underscore at the end. (i.e. ORKDynamicCast_)

And do not mark ORK_EXTERN because we don't want the symbol exposed when built as a framework. And drop ORK_AVAILABLE_DECL, which is not used in non-public headers.

It’s technically against the rules to use names that start with an underscore followed by a capital letter (C11 7.1.3p1)